assets/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="assets/css/style.css">
```

Comment:
However, if you keep the html file in the assets folder, this css url to "assets/css/style.css" will not work.  So you must either move index.html into root folder of your repostitory or make this href="css/style.css" ONLY MAKE ONE OF THESE CHANGES